src/client/game.cpp

```diff
@@ -4548,9 +4515,9 @@ void Game::showPauseMenu()
 		<< strgettext("Exit to Menu") << "]";
```

Comment:
the controls menu above here is still gated by `__ANDROID__`

Reply:
hmm, the controls / "change keys" menu could be useful even with `enable_touch=true` if you play with touchscreen (for camera movement) and keyboard (for everything else)

not sure why you'd do that though. we've already discussed this somewhere.

Reply:
I would prefer to not touch it at this moment.